app/src/main/java/eu/kanade/tachiyomi/util/view/ControllerExtensions.kt

```diff
@@ -775,6 +776,11 @@ fun Controller.requestFilePermissionsSafe(
             }
             .setNegativeButton(android.R.string.cancel, null)
             .show()
+    } else {
+        if (preferences.backupInterval().get() == 0 && !preferences.isBackupIntervalManuallyChanged().get()) {
```

Comment:
I think checking that the user never set a value should do the same
```suggestion
        if (preferences.backupInterval().isNotSet()) {
```